smithy-model/src/main/java/software/amazon/smithy/model/neighbor/RelationshipType.java

```diff
@@ -44,6 +44,13 @@
      */
     OPERATION("operation", RelationshipDirection.DIRECTED),
 
+    /**
+     * A collection operation relationship exists between a resource and the
+     * operations bound to the resource only in the "collectionOperations"
+     * property.
+     */
+    COLLECTION_OPERATION("collectionOperation", RelationshipDirection.DIRECTED),
```

Comment:
I think this is right, but the description doesn't line up with how it's implemented in NeighborVisitor.